src/components/media-views.js

```diff
@@ -388,6 +376,11 @@ AFRAME.registerComponent("media-video", {
       this.video.removeEventListener("pause", this.onPauseStateChange);
       this.video.removeEventListener("play", this.onPauseStateChange);
     }
+    if (this.hoverMenu) {
```

Comment:
Might not exist if somehow this `remove` is called before the `hoverMenu` promise resolves